src/components/outside/search/SeachPage.js

```diff
@@ -0,0 +1,37 @@
+import { useParams } from "react-router-dom";
+import { useSearchProduct } from "../../../hooks/useSearchProduct";
+import { Pagination } from "../../../utils/modules/Pagination/Pagination";
+import { ProductCard } from "../home/products/ProductCard";
+import { SearchPageWrapper } from "./SearchPage.styled";
+
+export const SearchPage = () => {
+  let { q } = useParams();
+  const { data } = useSearchProduct(q);
+  const { results: products, results_size } = data;
+
+  return (
+    <>
+      <SearchPageWrapper>
+        <div className="title">Results:</div>
+        {products && !results_size > 0 && (
```

Comment:
```suggestion
        { (products && !results_size > 0) && (
```
just to group that you need both conditions

Reply:
I think also that a better way of doing this is the following 
```suggestion
        {
          (products === [] || result_size < 0) <NoResults /> : <ProductList />
        }
```

try to alway reuse the components you already created and when you are creating them try to think on building them so they are easily reusable in other sections of the app